### PR TITLE
boosteroid 1.8.11 (new cask)

### PR DESCRIPTION
**Pull request by vnc0**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/b/boosteroid.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+    depends_on macos: ">= :catalina"
+  end
+  on_intel do
+    url "https://boosteroid.com/macos/installer/boosteroid-install-x64.dmg",
+        user_agent: :fake
+
+    depends_on macos: ">= :high_sierra"
+  end
+
```
Comment (khipp) [inline Casks/b/boosteroid.rb:17]:
```suggestion
  arch arm: "arm64", intel: "x64"
  folder = on_arch_conditional arm: "_ARM"

  version "1.8.11"
  sha256 :no_check

  url "https://boosteroid.com/macos#{folder}/installer/boosteroid-install-#{arch}.dmg"
```

File: Casks/b/boosteroid.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  name "Boosteroid"
+  desc "Cloud gaming service"
+  homepage "https://boosteroid.com/"
+
+  livecheck do
+    url "https://boosteroid.com/macos_ARM/client/changelog.md"
+    strategy :page_match do |page|
+      page.scan(/\[(\d+(?:\.\d+)+)\]/i).map { |match| match[0] }
+    end
```
Comment (khipp) [inline Casks/b/boosteroid.rb:26]:
```suggestion
    regex(/\[\s*\v?(\d+(?:\.\d+)+)\s*\]/i)
```

File: Casks/b/boosteroid.rb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  end
+
```
Comment (khipp) [inline Casks/b/boosteroid.rb:28]:
```suggestion

  on_arm do
    depends_on macos: ">= :catalina"
  end
  on_intel do
    depends_on macos: ">= :high_sierra"
  end

```

**Review by khipp — COMMENTED** [review]
Thank you for your submission, @vnc0!

File: Casks/b/boosteroid.rb (hunk 1 above)
Comment (vnc0) [reply Casks/b/boosteroid.rb:17]: Thank you! I will incorporate the suggestions. Changing the user agent `user_agent: :fake` is necessary though.

File: Casks/b/boosteroid.rb [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  version "1.8.11"
+  sha256 :no_check
+
+  on_arm do
+    depends_on macos: ">= :catalina"
+  end
+  on_intel do
+    depends_on macos: ">= :high_sierra"
+  end
```
Comment (p-linnane) [inline Casks/b/boosteroid.rb:13]:
```suggestion
  depends_on macos: ">= :high_sierra"
```

Comment (p-linnane) [reply Casks/b/boosteroid.rb:13]: arm64 begins with Big Sur, so scoping this just to High Sierra+ is fine.

**Review by p-linnane — DISMISSED** [review]

**Review by khipp — APPROVED** [review]

**Review by bevanjkay — APPROVED** [review]

**Pull request merged**